plugins/postcss-custom-properties/src/index.ts

```diff
@@ -86,7 +89,11 @@ const creator: PluginCreator<PluginOptions> = (opts?: PluginOptions) => {
 					Declaration: (decl) => {
 						transformProperties(decl, customProperties, { preserve });
 					},
-					OnceExit: () => {
+					OnceExit: (root, { result }) => {
+						if (!disableDeprecationNotice && (importFrom.length > 0 || exportTo.length > 0)) {
+							root.warn(result, '"importFrom" and "exportTo" will be removed in a future version of postcss-custom-properties.\nCheck the discussion on github for more details. https://github.com/csstools/postcss-plugins/discussions/192');
```

Comment:
Should we also encourage to weigh in on the discussion?

Reply:
Good point!

I initially tried to keep these brief.
But then I added the flag to silence them.

Something like this :

> "importFrom" and "exportTo" will be removed in a future version of postcss-custom-properties.
> We are looking for insights and anecdotes on how these features are used so that we can design the best alternative.
> Please let us know if our proposal will not work for you.
> Visit the discussion on GitHub for more details. https://github.com/csstools/postcss-plugins/discussions/192